game_app.rb

```diff
@@ -0,0 +1,93 @@
+require_relative 'game'
+require_relative 'author'
+
+class GameApp
+  def initialize
+    @authors = []
+    @games = []
+  end
+
+  def list_authors
+    if @authors.empty?
+      puts 'No authors found'
+    else
+      puts 'Authors: '
+      @authors.each { |author| puts "Fullname: #{author.last_name} #{author.first_name} " }
+    end
+  end
+
+  def list_games
+    puts 'No games found.' if @games.empty?
+    @games.each do |game|
+      puts "Published on : #{game.publish_date}"
+      puts "Last Played on : #{game.last_played_at}"
+      puts "Multiplayer? : #{game.multiplayer}"
+      puts '.....................'
+    end
+  end
+
+  def create_game(sel, publish_date, multi, last_played)
+    author_ind = sel - 1
```

Comment:
- [ ] [Optional] It would be better to name your parameters as descriptive as possible so that anyone can understand your code easily.